ksqldb-engine/src/test/java/io/confluent/ksql/ddl/commands/CreateSourceFactoryTest.java

```diff
@@ -911,6 +929,41 @@ public void shouldThrowIfTableIsMissingPrimaryKey() {
         containsString("Tables require a PRIMARY KEY. Please define the PRIMARY KEY."));
   }
 
+  @Test
+  public void shouldThrowIfStreamExits() {
```

Comment:
Could you add two more tests shouldNotThrowWhenCreateStreamIfNotExistsIsSet and same for a table?